src/replicate.cpp

```diff
@@ -47,15 +47,16 @@ void Replicate::command(int narg, char **arg)
   int me = comm->me;
   int nprocs = comm->nprocs;
 
-  if (me == 0) utils::logmesg(lmp,"Replicating atoms ...\n");
-
   // nrep = total # of replications
 
   int nx = utils::inumeric(FLERR,arg[0],false,lmp);
   int ny = utils::inumeric(FLERR,arg[1],false,lmp);
   int nz = utils::inumeric(FLERR,arg[2],false,lmp);
   int nrep = nx*ny*nz;
 
+  if (me == 0)
+    utils::logmesg(lmp,"Replication is creating a {}x{}x{} = {}x larger system...\n",nx,ny,nz,nrep);
```

Comment:
sorry, though better of {}x larger, so instead:
= {} times larger system

Reply:
done as well